mobile-app/app/screens/AppNavigator/screens/Loans/components/PriceOracleInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import { tailwind } from "@tailwind";
+import { ThemedIcon, ThemedTextV2 } from "@components/themed";
+import { translate } from "@translations";
+import { View } from "react-native";
+import { TouchableOpacity } from "react-native-gesture-handler";
+
+interface PriceOracleInfoProps {
+  onPress: () => void;
+}
+export function PriceOracleInfo(props: PriceOracleInfoProps): JSX.Element {
+  return (
+    <View style={tailwind("items-center")}>
+      <TouchableOpacity
+        onPress={props.onPress}
+        style={tailwind("text-center flex-row items-center")}
+      >
+        <ThemedTextV2
+          dark={tailwind("text-mono-dark-v2-500")}
+          light={tailwind("text-mono-light-v2-500")}
+          style={tailwind("font-normal-v2 text-sm mr-1")}
+        >
+          {translate("", "Loan tokens get their prices from oracles.")}
```

Comment:
<img width="532" alt="image" src="https://user-images.githubusercontent.com/13292662/193224813-a9e2fef4-7056-489c-bfae-1e85ebd69d91.png">
should be outside dark mode background

Reply:
translations missing?